SDK update for 2023-06-01-preview

matthohn-msft opened the pull request:
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

amnguye committed:
Prepare for WebJobs Storage Extension Release (#38943)

kinelski committed:
[AppConfiguration] Fix: GetConfigurationSettings does not set the ContentType property (#38917)

cochi2 committed:
Adding Hold/Unhold capabilities to ACS. (#38858)

* Adding hold/unhold

* Updating swagger

* Fixing last merge

* Adding tests and missing apis

* Exporting new APIs

* Updating swagger

* Adding missing files

* Updating docstring

* Export API

* Updating swagger location

azure-sdk committed:
Update AutoRest C# version to 3.0.0-beta.20230924.2 (#38934)

* Update Generator Version 3.0.0-beta.20230924.2

* Update SDK codes

* remove unused Etag customization

---------

Co-authored-by: FAREAST\chunyu <chunyu@microsoft.com>

HarveyLink committed:
[Network] Fix partial resource (#38110)

azure-sdk committed:
Increment package version after release of Azure.ResourceManager.DataProtectionBackup (#38953)

azure-sdk committed:
Update Generator Version 3.0.0-beta.20230925.2 (#38955)

azure-sdk committed:
Increment version for storage releases (#38946)

* Increment package version after release of Microsoft.Azure.WebJobs.Extensions.Storage.Blobs

* Increment package version after release of Microsoft.Azure.WebJobs.Extensions.Storage.Queues

* Increment package version after release of Microsoft.Azure.WebJobs.Extensions.Storage

azure-sdk committed:
Increment package version after release of Azure.ResourceManager.Network (#38957)

azure-sdk committed:
Increment package version after release of Azure.ResourceManager.RecoveryServicesSiteRecovery (#38958)

azure-sdk committed:
Increment package version after release of Azure.ResourceManager.Chaos (#38956)

frantran committed:
Fleet .NET SDK generation (#38873)

azure-sdk committed:
Increment package version after release of Azure.ResourceManager.DatabaseFleetManager (#38960)

jaschrep-msft committed:
DataMovement Test Infra: In-Memory StorageResources (#38941)

* memoryresources initial implementation

* revised implementaoin

* testing memoryitem

* test container

Jingshu918 committed:
[DataFactory]Upgrade SDK Version To 1.0.0-beta.4 (#38930)

azure-sdk committed:
Increment package version after release of Azure.ResourceManager.DataFactory (#38967)

dvbb committed:
[Network] Fix HopLink format error issue when resourceId return "" (#38876)

kinelski committed:
[FormRecognizer] Cleaning up misc operations live tests (#38807)

benbp committed:
Update messaging stress to use addons parallel job configuration (#38949)

rajkumar-rangaraj committed:
Vendor OpenTelemetry.NET Instrumentation Libraries into Azure.Monitor.OpenTelemetry.AspNetCore (#38848)

* Vendoring

* Update copyrights

* Update Notice file

rajkumar-rangaraj committed:
Remove the default OpenTelemetry instrumentation options customization from the README. (#38970)

rajkumar-rangaraj committed:
Remove OpenTelemetry ASP.NET Core and HTTP metrics. (#38972)

jaschrep-msft committed:
DataMovment `StorageResourceContainer` internal hooks to protected APIs (#38968)

* internals access class and hookup to file

* applied to blobs

* docsting

amnguye committed:
Implemention ShareFileStorageResource (#38938)

* WIP

* Implemented base file storage resource; Testst

* Export API

* Attempt to fix double dependency files in tests

* Addressing PR comments; Added StorageResourceItemInternal

* Rename internal methods

* PR Comments for tests

* CHange out verify parameters with  the mock verify method

hallipr committed:
Support typespec to sdk preview pipeline (#38576)

* Add changes to support typespec preview pipeline

* Revert whitespace changes

* Improve error logging in Update-dotnet-GeneratedSdks

scottaddie committed:
Update Monitor labels (#38974)

azure-sdk committed:
Change prod maintenance day. Fix stress watcher deploy (#38948)

Co-authored-by: Ben Broderick Phillips <bebroder@microsoft.com>

rohantagaru committed:
Azure Deployment Manager is being decommissioned. Remove its NET SDK through this PR (#38614)

azure-sdk committed:
Update Generator Version 3.0.0-beta.20230926.2 (#38979)

azure-sdk committed:
Update AutoRest C# version to 3.0.0-beta.20230927.1 (#38987)

m-redding committed:
fix (#38995)

jsquire committed:
[AppConfig] Allow unknown condition members (#38971)

* [AppConfig] Allow unknown condition members

The focus of these changes is to allow for unknown members to be
present in the `conditions` property of a feature flag configuration
setting.  Previously, these were ignored as the setting assumed that
`conditions` should be treated as a well-known member.  This change moves
the well-known member to `conditions/client_filters` and preserves unknown
children of `conditions`.

seanmcc-msft committed:
Enabled STG 91 live tests (#38990)

m-redding committed:
[Core] Adding attributes to support Native AOT compiling (#37734)

* part 1

* additional changes

* additional attributes

* additional attributes

* adding diagnosticscope attributes

* updating messages

* more fixes

* additional updates

* updates

* PR feedback

* API update

* removing unnecessary warnings

* reverting attribute on HttpPipelineSynchronousPolicy

* update api

* updates

* update

* API export

* WIP fix 1

* WIP

* attribute adjustments

* adding compatibility test

* WIP adding test

* WIP API

* WIP

* WIP

* updates

* add test script and remove #if statements

* fixing build

* Update Program.cs

* adding some compile statements

* Update AzureCoreEventSource.cs

* build fix attempt

* fix built attempt 2

* Update MutableJsonChange.cs

* Update MutableJsonChange.cs

* Update MutableJsonDocument.cs

* Update HttpPipelineSynchronousPolicy.cs

* adding comments

* PR fb 1

* test

* test2

* test3

* attempt

* additional build fixes

* Remove comment

* PR feedback + fixing impacts of feedback changes

* fixes

* Update GeoRedundantFallbackPolicy.cs

* Update GeoRedundantFallbackPolicy.cs

* Update GeoRedundantFallbackPolicy.cs

* fix

* add tests

* pipeline fixes

* update comments

* feedback

* adjustments

* adjustment

* pipeline fix

* PR feedback 1

* Apply suggestions from code review

Co-authored-by: Vitek Karas <10670590+vitek-karas@users.noreply.github.com>

* PR feedback updates 2

* Update sdk/core/Azure.Core/src/Shared/DiagnosticScope.cs

Co-authored-by: Vitek Karas <10670590+vitek-karas@users.noreply.github.com>

* need one other suppression

* remove 2 annotations + update suppression message

* removing all attributes from RequestContentHelper

* tweak justification

* removing test project from Azure.Core, will be adding feedback to general CI pipeline

* WIP

* feedback

* Update sdk/core/Azure.Core/src/Shared/DiagnosticScope.cs

Co-authored-by: Anne Thompson <annelo@microsoft.com>

* fix attribute and add comments

* Addressing feedback

* adding net 5 preprocessor directives

* Update sdk/core/Azure.Core/src/DynamicData/MutableJsonElement.cs

Co-authored-by: Vitek Karas <10670590+vitek-karas@users.noreply.github.com>

* remove unused const

* pipeline fix

---------

Co-authored-by: Vitek Karas <10670590+vitek-karas@users.noreply.github.com>
Co-authored-by: Anne Thompson <annelo@microsoft.com>

rangothedog committed:
[Text Translation] Options Parameter for Translate and Transliterate (#38760)

* Options Parameter for Translate and Transliterate

* Remove empty options constructor, add new samples and recorded tests, update documentation.

* Update CHANGELOG.md and assets.json, fixed bug in TranslationLiveTests.cs

* Deleting unnecessary SessionRecords test files.

---------

Co-authored-by: Rango Meadows <v-rameadows@microsoft.com>

pshao25 committed:
Add rules for convenience method and protocol method (#38603)

* Add rules for convenience method and protocol method

* update

azure-sdk committed:
Increment version for managednetworkfabric releases (#39010)

azure-sdk committed:
Update AutoRest C# version to 3.0.0-beta.20230927.3 (#39013)

jaschrep-msft committed:
ShareDirectoryClient extensions (#38993)

* ShareDirectoryClient extensions

* removed mention of blobs

* exportapi

* fix tests

azure-sdk committed:
Update AutoRest C# version to 3.0.0-beta.20230928.1 (#39017)

vivekmore-msft committed:
Added transcription APIs and events (#38988)

* Autorest generated code

* Adding SDK methods for start, stop and update transcription

* Add transcription options for the create & answer call

* Adding events

* Adding event parser test cases

* Adding tests

* Refactoring code to make options param optional

* Adding models and parser for the transcription schema

* Fixing enum conversion logic for transcription schema

* Cleaning up code

* Add transcription options in create/answer/group call and tests

* Updted TranscriptionUpdate to take TranscriptionUpdate and TranscriptionStatusDetails as enums

* Fixed API typo

* Adding SDK methods for start, stop and update transcription

* Add transcription options for the create & answer call

* Adding models and parser for the transcription schema

* Adding events

* Adding event parser test cases

* Adding tests

* Refactoring code to make options param optional

* Fixing enum conversion logic for transcription schema

* Cleaning up code

* Add transcription options in create/answer/group call and tests

* Updted TranscriptionUpdate to take TranscriptionUpdate and TranscriptionStatusDetails as enums

* Fixed API typo

* Updated contract after rebase

* Removed duplicate tests

* Update in transcription ws schema

* Fixed TranscriptionOptions review comment

* Fixing TranscriptionOptions arguments in create call, group call & answer call

* Fixing build issues

---------

Co-authored-by: abhishesingh-msft <abhishesingh@microsoft.com>

seanmcc-msft committed:
Modified storage ARM template to point at built-in Files RBAC role (#39018)

JoshLove-msft committed:
Add support for settlement from the isolated worker extension (#38865)

* Add support for settlement from the isolated worker extension

* Update comment

* roll back version of Grpc.Tools

* roll back further

* PR fb

* Revert eventArgs fields to private

* remove in finally block

* Fix test and move versions to package.data.props

* Update sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Listeners/ServiceBusListener.cs

Co-authored-by: Jesse Squire <jesse.squire@gmail.com>

* Add batch test cases

* Fix tests

* Fix

---------

Co-authored-by: Jesse Squire <jesse.squire@gmail.com>

ArthurMa1978 committed:
Update the migration guide. (#38961)

glecaros committed:
[ARM] Updating Azure.ResourceManager.MachineLearning to generate for … (#38814)

azure-sdk committed:
Increment package version after release of Azure.ResourceManager.MachineLearning (#39035)

azure-sdk committed:
Increment package version after release of Azure.ResourceManager.RecoveryServicesDataReplication (#39034)

JimSuplizio committed:
add traling slash to directory wildcard entries (#39038)

JimSuplizio committed:
Update CODEOWNERS (#39039)

Just testing to see if this path isn't recognizing owners because the ones there were invalid

weshaggard committed:
Revert "Update CODEOWNERS (#39039)" (#39040)

This reverts commit ec8e5bb1cfb86d8824de5ac3c3973dc1e920fd1c.

benbp committed:
Remove old check enforcer comment template (#39042)

jsquire committed:
[JimBot] Remove CXP Attention (#39036)

Removing config for the dead CXP Attention rule

christothes committed:
Multi-tenant tests added as recorded tests (#38992)

christothes committed:
TokenExchangeManagedIdentitySource with async IO (#38939)

heaths committed:
Add AOT target for Key Vault SDK (#38721)

* Add AOT target for Key Vault SDK

Resolves #38500

* Rebase on main, resolve PR feedback

rajkumar-rangaraj committed:
Detect Instrumentation Package and Backoff  (#39001)

* AddVendorInstrumentationIfPackageNotReferenced

* PR feedback.

* event souce fix.

amnguye committed:
Replace LocalFilesStorageResourceProvider.FromPath with FromFile and Directory (#39053)

* Replace LocalFilesStorageResourceProvider.FromPath with FromFile and FromDirectory

* Export API

* Update snippets

azure-sdk committed:
Sync eng/common directory with azure-sdk-tools for PR 7040 (#39052)

* Remove codeowners resolution logic from docs scripts

* Remove parameters that aren't needed

* Remove auth information from update-docsms-metadata.yml

---------

Co-authored-by: Daniel Jurek <djurek@microsoft.com>

heaths committed:
Add support for Key Vault 7.5-preview.1 (#38377)

* Add support for Key Vault 7.5-preview.1

Also treats KeyRotationPolicyAction as case-insensitive.

* Update public APIs

* Use latest commit

Resolves #38729

Commit:
[AzureMonitorExporter] resolve AOT warnings (#38459)

* initial commit

* build fix

* workaround for StackFrame.GetMethod()

* cleanup

* fix test

* remove TrimmingAttribute.cs

* temp disable ApiCompat

* cleanup

* test fix for ApiCompat

* update comment

* add aotcompat test app

* add readme

* readme

* recommended fix for ApiCompat

* comment

* test fix for validation errors re: aotcompat directory

* isolate StackFrame.GetMethod

* cleanup

* isolate StackFrame.GetMethod (2)

* add comment.

* fix

* pr feedback

* update comment

* fix script

* refactor as extension method

* cleanup

abhishesingh-msft committed:
Transcription package fix: (#39066)

1> Made non-internal TranscriptionData & Word classes as public
2> Rename Word to WordData, as single word classes are not allowed
3> Cleanup vars & references

jaschrep-msft committed:
removed mock resource types (#39020)

Now uses internal types elsewhere in the inheritance line

danieljurek committed:
Remove AAD credentials from Service-Level-Readme-Automation.ps1 invocation (#39061)

jalauzon-msft committed:
[Storage][DataMovement] Update checkpointer to read/write to job file - Part 1 (#39057)

amnguye committed:
[DataMovement] Adding Tests for Recorded File Upload (Add fix Creating File at Initial Start) (#39037)

* Added Single Share File Upload Tests; Create before putrange

* WIP

* Removed testbase for startTransfer upload test base; other pr comments

rajkumar-rangaraj committed:
Vendor OpenTelemetry.ResourceDetectors.Azure resource detector package. (#39076)

richardcho-msft committed:
Richardcho/cancel live test (#38969)

* add live test

* record test

* ignore test

---------

Co-authored-by: root <root@DESKTOP-6GBNLER>

rajkumar-rangaraj committed:
Update Changelog/Readme for Azure.Monitor.OpenTelemetry.AspNetCore (#39077)

* Update Changelog

* Remove forked.

* Update Readme

* pr feedback

* Update API Comment.

* Update changelog

ShivangiReja committed:
[Search] Regenerate code using 2023-10-01-preview API version (#38925)

* Regenerate code

rajkumar-rangaraj committed:
Replace Azure.Monitor.OpenTelemetry.Exporter project reference with Package Reference. (#39082)

calvinkwtang committed:
Removed unnecessary .Models.Events namespace (#39075)

mikehang-msft committed:
Added Support for PSTN Dial-Out Feature For ACS Rooms (.NET) SDK  (#38837)

* Updated ACS Rooms .NET SDK to Support PSTN DialOut Feature

* Removed Auto Script Generation Snippet From Older Test Class

* Updated Readme Link

* Updated new tag in Rooms assets.json

* Updated ChangeLog and Generated new export API

* Modified UpdateRoom Signagure and Renamed Variables

* Updated ReadMe and added more test scenarios

* Update ReadMe

* Update ReadMe with Snippets

* Updated Rooms Sample

* Updated Codes Per SDK Reviews

* Updated ChangeLog

rajkumar-rangaraj committed:
Fix the assembly load that validates the package reference. (#39091)

rajkumar-rangaraj committed:
[AzureMonitorDistro] Update the troubleshooting section in the Readme (#39096)

* Update Readme

* pr feedback

* pr feedback

rajkumar-rangaraj committed:
Prepare new release: Distro beta8 (#39100)

vivekmore-msft committed:
Ignoring case (#39105)

christothes committed:
Prep for release (#39108)

ShivangiReja committed:
[Search] Update changelog for "11.5.0-beta.5" release (#39102)

[Search] Update changelog for "11.5.0-beta.5" release.

nisha-bhatia committed:
[Azure.Monitor.Ingestion] Fix Sovereign Support for Ingestion (#39078)

ShivangiReja committed:
[App Config] Update snapshot API and changelog for "1.3.0-beta.3" release (#39116)

 [App Config] Update snapshot API and changelog for "11.5.0-beta.5" release

azure-sdk committed:
Updating package version for configured workflows (#39119)

Co-authored-by: Jesse Squire <jesse.squire@gmail.com>

ShivangiReja committed:
[Search] Add a work around to fix CreateAzureBlobIndexer test (#39117)

amnguye committed:
Renamed Storage PublicAudience to DefaultAudience, Get..Audience to Create..Audience (#39083)

azure-sdk committed:
Increment package version after release of Azure.Monitor.OpenTelemetry.AspNetCore (#39129)

sofiar-msft committed:
Add livetests to PPE (#39110)

* Update tests.yml

* Skip SIP tests on int and ppe

* Skip SIP tests on ppe

* Add flag to skip sip configuration tests

* Add flag to skip sip configuration in matrix

* Update to net462

* Remove Siprounting flag

* Remove Siprounting flag from matrix

* Add Siprounting flag to matrix

azure-sdk committed:
Detect all file diff types for eng/common changes (#39128)

Fixes https://github.com/Azure/azure-sdk-tools/issues/5882

We need to set the difffilter to empty instead of the default of exclude deleted files when we are trying to verify there are no changes under eng/common.

See test PR https://github.com/Azure/azure-sdk-for-python/pull/32348 which demonstrates us not detecting a deleted file under eng/common. I'll use that same test PR to verify this now catches that issue.

Co-authored-by: Wes Haggard <weshaggard@users.noreply.github.com>

cochi2 committed:
Adding CorrelationId to TranscriptionMetadata (#39130)

mikehang-msft committed:
Update ChangeLog file (#39125)

jalauzon-msft committed:
[Storage][DataMovement] Update checkpointer to read/write to job file - Part 2 (#39101)

ArthurMa1978 committed:
Release Azure.ResourceManager.ContainerServiceFleet (#39106)

* Release Azure.ResourceManager.ContainerServiceFleet

* update

azure-sdk committed:
Increment package version after release of Azure.ResourceManager.ContainerServiceFleet (#39139)

ArcturusZhang committed:
fix pageable operation issue in purview administration library (#39009)

live1206 committed:
use directive to set api-version as client for Resources_GetById (#39141)

azure-sdk committed:
Update AutoRest C# version to 3.0.0-beta.20231008.1 (#39143)

* Update Generator Version 3.0.0-beta.20231008.1

* Update SDK codes

* Update SDK codes

* Update SDK codes

* Update SDK codes

* Update SDK codes

* Update SDK codes

azure-sdk committed:
Update Generator Version 3.0.0-beta.20231008.2 (#39148)

zackliu committed:
[WebPubSubClient] Parsing message returns IList (#39137)

* Parsing message returns IList

* update api

* Use IReadonlyList

nisha-bhatia committed:
Update CHANGELOG.md (#39160)

azure-sdk committed:
Increment package version after release of Azure.Search.Documents (#39161)

nisha-bhatia committed:
[Azure.Monitor.Inestion] Update CHANGELOG.md for GA Release (#39165)

wiboris committed:
Adding Support to override the ClientRequestID guid format for test recording sessions. (#39133)

* adding an enviroment variable "RECORDING_DEFAULT_ClIENT_GUID" to support using the default string format for ClientID's in test recording mode

* changing code to support storing the enviroment variable into the test recordings variables

* handling the case if the variable is not present in the recording

* changing from enviroment var to property of RecordedTestBase

* removing files

* whitespace fix

* update

* recovered deleted file

* restored deleted code

* fix

* feedback

(cherry picked from commit 240efa7ad801712117eb3624791431f92aa132d6)

* removed dead code

* whitespace

live1206 committed:
Move ResourcesRestOperations to generated (#39150)

* Move most of ResourcesRestOperations to generated

* make api-version as method parameter for Resources_GetById

* Add comments

azure-sdk committed:
Increment package version after release of Azure.Data.AppConfiguration (#39169)

azure-sdk committed:
Increment package version after release of Azure.Monitor.Ingestion (#39168)

azure-sdk committed:
Update Generator Version 3.0.0-beta.20231009.1 (#39173)

glorialimicrosoft committed:
Add azure communication messages sdk (#39167)

* Add Messages SDK

* Work around the issue with nextLink never empty, add more tests

* Update CHANGELOG.md

* Update autorest and use Github link for input file

* Update README.md

* dropping the Value suffix on everything but the base class (MessageTemplateValue)

* change the param name from url to uri, filename to fileName

* Update CHANGELOG.md

* revert accidental check in of RecordedTestMode

* sanitize session records

* remove unneeded import

* update to 2023-08-24-preview api version

* update CommunicationMessagesClientOptions api version to 2023-08-24-preview

* re-run live test recording

* fix broken link

* use latest swagger

* Remove unsupported auth type, add more live tests, and re-run the live tests

* use the swagger that does not have operation-id response header

* Take Value suffix off MessageTemplateQuickAction

* update sdk to latest

- change autorest to using readme instead of swagger input-file
- update Butotn from IDictionary to IEnumerable for maintaining orders
- autorest point to azure-rest-api-specs-pr
- re-run live tests to get new session records

* update the source file to the public azure specs repo

* fix recording

* run "eng\scripts\Export-API.ps1 communication"

* run "eng\scripts\CodeChecks.ps1 -ServiceDirectory communication"

JoshLove-msft committed:
Prepare for release (#39184)

scottaddie committed:
Replace Azure.Identity AAD references (#39123)

jalauzon-msft committed:
[Storage][DataMovement] Add ProviderId to resources and job plan file (#39163)

azure-sdk committed:
Update github-event-processor version to 1.0.0-dev.20231010.2 (#39189)

Co-authored-by: James Suplizio <jasupliz@microsoft.com>

vivekmore-msft committed:
Fixed tansxription review comments (#39195)

azure-sdk committed:
Increment package version after release of Azure.Communication.Messages (#39185)

JoshLove-msft committed:
Fix broken link (#39188)

JoshLove-msft committed:
Prepare for release (#39203)

azure-sdk committed:
Increment package version after release of Microsoft.Azure.WebJobs.Extensions.EventHubs (#39201)

JoshLove-msft committed:
Add new system events (#39202)

* Add new system events

* Fix source generator

* refactor

JoshLove-msft committed:
Prepare for release (#39205)

azure-sdk committed:
Use url version spec for autorest.* packages (#39131)

Co-authored-by: Patrick Hallisey <pahallis@microsoft.com>

azure-sdk committed:
Increment package version after release of Azure.Messaging.ServiceBus (#39204)

mcgallan committed:
Fix OperationalInsights Type Issue (#38425)

* update

* update

m-redding committed:
[Core] Update RequestFailedException to use source generation for serialization (#38996)

* iteration 1

* iteration 1

* tweak logic

* simplification

* adjust comments

* Update sdk/core/Azure.Core/src/RequestFailedException.cs

Co-authored-by: Eric Erhardt <eric.erhardt@microsoft.com>

* feedback fixes

---------

Co-authored-by: Eric Erhardt <eric.erhardt@microsoft.com>

JoshLove-msft committed:
Rename property in ResourceNotificationsResourceUpdatedDetails (#39210)

azure-sdk committed:
Use better description for single repo sparse checkout (#39209)

Co-authored-by: Patrick Hallisey <pahallis@microsoft.com>

scottaddie committed:
Document sovereign cloud support in LogsIngestionClient (#39221)

JoshLove-msft committed:
Dispose clients on host shutdown rather than listener dispose (#39225)

* Dispose clients on host shutdown rather than listener dispose

* Remove project reference

* Update version - for some reason the auto-update job failed

azure-sdk committed:
Update Generator Version 3.0.0-beta.20231012.5 (#39239)

amariwest-msft committed:
Add PauseOnStart Recording Option  (#39132)

* generated files using swagger

* removed extra swagger file

* modified unit tests

* undo status code change for mute participant in generated files

* merged with main

* merged with main

* reverted 202 change

* Both status code changed to 202

* modified test cases to check for 200 status code

* regenerated code

* regenerated code

* Revert "regenerated code"

This reverts commit a5a9e0e5b8296fbf90a49dc8ebacd62f75298690.

* Added TransciptionDataRequest Change

* generated from latest swagger

* generated netstandard2.0.cs

* edited changelog

* Revert "generated netstandard2.0.cs"

This reverts commit 2a39b2d865a5a223e0edaf7e3615dc16562e85b7.

* Revert "edited changelog"

This reverts commit 7eb14250f0a123983d82e606ed8c7fb5d4585a86.

* added line to changelog and standard2.0.cs

* Update CHANGELOG.md

audunn committed:
[NetAppFiles] Update to api-version 2023-05-01 (#38882)

* Update to api-version 2023-05-01

* SystemData

* Sync + systemdata

* update assets

* update assets

* s2ync

* EB.Never

* EB.Never missing using ??

* export api

* change changelog

* assets

* assets

* assets

frantran committed:
add autorest changes to support fleet issues (#39172)

azure-sdk committed:
Increment package version after release of Azure.ResourceManager.ContainerServiceFleet (#39256)

azure-sdk committed:
Increment package version after release of Azure.Messaging.EventGrid (#39262)

JoshLove-msft committed:
Refactor cleanup (#39264)

JimSuplizio committed:
Fix CODEOWNERS for missing paths and other fun stuff (#39269)

* Fix CODEOWNERS for missing paths and other fun stuff

* Fix the label entry, without the % it isn't recognized as a label

jalauzon-msft committed:
[Storage][DataMovement] Fix several issues with pause/resume (#39208)

nisha-bhatia committed:
[Azure.Monitor.Ingestion] Update CHANGELOG.md (#39234)

pallavit committed:
Update code owners list in the .NET repo (#39180)

* Update codeowners list.

azure-sdk committed:
Autorest Regen Preview to 3.0.0-alpha.20231013.8 by Michael Nash from refs/pull/3810/head (#39275)

* Update Generator Version 3.0.0-alpha.20231013.8

* Update SDK codes he_ma_3

* Update SDK codes da_he_2

* Update SDK codes me_pu_4

* customize the access for ImageSize

* update devcenter api

* update custom code to use new reponse type

* customize deserialization for lro return type change

* Update Generator Version 3.0.0-beta.20231013.1

---------

Co-authored-by: m-nash <prognash@gmail.com>

yaotongms committed:
Release stable version 2022-12-27 of HybridCompute (#39217)

* release stable version

* update code

* update code

* update code

azure-sdk committed:
Increment package version after release of Azure.Communication.Rooms (#39231)

m-redding committed:
[Core] Add `DiagnosticScope` test and removing bug workarounds that are no longer needed (#39274)

* new tests

* formatting

m-redding committed:
[Schema registry] Stream fix / typo (#39270)

* fixes

* Update sdk/schemaregistry/Azure.Data.SchemaRegistry/samples/Sample02_Serialization.md

Co-authored-by: Jesse Squire <jesse.squire@gmail.com>

---------

Co-authored-by: Jesse Squire <jesse.squire@gmail.com>

jabberwik committed:
Makes AzureKeyVaultConfigurationSource a public class (#39099)

* Makes AzureKeyVaultConfigurationSource a public class

* Updates generated code

jsquire committed:
[Extensions]: Update KeyVault Config change log (#39292)

The focus of these changes is to update the change log for the KeyVault configuration package to acknowledge the recent contribution for exposing the config section as public.

azure-sdk committed:
Set product as parent of package work item. (#39156)

Co-authored-by: praveenkuttappan <prmarott@microsoft.com>

sarkar-rajarshi committed:
[ACS JobRouter] DPG Models (#38868)

* missing usage of Azure.Etag in RouterQueueItem

* add tsp-location

* dpg models initial commit

* update commit

* all Serialize methods added

* auth policy modified to new generated code

* fix errors and update readme

* fix tests

* Update with more fixes

* more fixess

* update assets

* update assests.json

* update api def

* more fixes

seanmcc-msft committed:
STG 91 beta release (#39297)

nisha-bhatia committed:
[Azure.Monitor.Ingestion] update Changelog for release (#39304)

azure-sdk committed:
Sync eng/common directory with azure-sdk-tools for PR 7113 (#39226)

* Remove ARM deployment after deploy

* Update eng/common/TestResources/New-TestResources.ps1

Co-authored-by: Heath Stewart <heaths@outlook.com>

* Apply suggestions from code review

---------

Co-authored-by: Ben Broderick Phillips <bebroder@microsoft.com>
Co-authored-by: Heath Stewart <heaths@outlook.com>

trrwilson committed:
fix Azure OpenAI whisper snippets (#39313)

live1206 committed:
Prepare release for hybridcompute (#39318)

azure-sdk committed:
Increment package version after release of Azure.ResourceManager.HybridCompute (#39320)

azure-sdk committed:
Increment version for storage releases (#39302)

jalauzon-msft committed:
[Storage][DataMovement] More fixes to resuming transfers (#39271)

JoshLove-msft committed:
Use bytes to represent AMQP property map (#39307)

* Use bytes to represent AMQP property map

* Delete SettlementPropertiesTests.cs

* PR fb

* Fix test and revert testbase change

* remove unused import

azure-sdk committed:
Update AutoRest C# version to 3.0.0-beta.20231017.1 (#39329)

* Update Generator Version 3.0.0-beta.20231017.1

* Update SDK codes me_pu_4

JoshLove-msft committed:
Use singleton rather than hosted service for cleanup (#39327)

* Use singleton rather than hosted service for cleanup

* prepare for release

* add date

seanmcc-msft committed:
Added x-ms-file-request-itent to PutRangeFromUrl (#39278)

Commit:
Merge identity/1.10.2-rel back to main (#39331)

* Bug fixes for development time credentials

* update project version

* Increment package version after release of Azure.Identity (#39170)

* tests

---------

Co-authored-by: Azure SDK Bot <53356347+azure-sdk@users.noreply.github.com>

nisha-bhatia committed:
[Azure.Monitor.Query] Add MetricsBatch (#38640)

frantran committed:
add ContainerServiceFleet test support (#39348)

audunn committed:
[NetAppFiles] Update version number to stable (#39352)

* Update version number to stable

* Update release date

azure-sdk committed:
Increment package version after release of Microsoft.Azure.WebJobs.Extensions.ServiceBus (#39339)

azure-sdk committed:
Increment package version after release of Azure.Monitor.Ingestion (#39334)

abatishchev committed:
Redo (#39060)

jiriburant committed:
Fixing communication test scripts (#39355)

* Updated test-resources template to contain necessary properties.

* Added logging for test environment variables for better troubleshooting.

christothes committed:
Azure.Identity.Broker archboard feedback (#39296)

JoshLove-msft committed:
Fix namespace of ACS models (#39341)

* Fix namespace of ACS models

* Fix

amnguye committed:
[Storage] [DataMovement] Added Download File Share Tests and Minor Fixes (#39206)

* Added download tests for file shares

* Rerecorded tests

* Cleanup

JoshLove-msft committed:
Apply EBN to back compat ACS models (#39360)

jaschrep-msft committed:
DataMovement unify TransferAndVerify Part 1  (#39300)

* initial unified implementation

* applied to blob single download tests

* move local tooling

* checkpoint

* separate out blob utilities

* separating item and container test util

* cancellation tokens

* using single blob

* apply to append tests

* added page blobs

* remove old verify methods

rajkumar-rangaraj committed:
Fix: Prevent data duplication in TraceTelemetry with IncludeFormattedMessage (#39308)

* Fix: Prevent data duplication in TraceTelemetry with IncludeFormattedMessage.

* Update Changelog

* PR feedback on Exception original format.

* test changes

* Test changes.

abhishesingh-msft committed:
Implement generci parser for Transcription, Audio data (#39250)

JoshLove-msft committed:
One more EBN and add test (#39365)

* One more EBN and add test

* Fix

JoshLove-msft committed:
Fix proto definition (#39361)

christothes committed:
prep for Identity patch release (#39359)

nisha-bhatia committed:
Update CHANGELOG.md (#39343)

azure-sdk committed:
Increment package version after release of Microsoft.Azure.WebJobs.Extensions.ServiceBus (#39369)

azure-sdk committed:
Increment package version after release of Azure.Identity (#39374)

azure-sdk committed:
Increment package version after release of Azure.Monitor.Query (#39372)

chunyu3 committed:
add a funtion to convert BinaryData to RequestContent (#39316)

* add serialize funtion for BinaryData

* add more test

* Update sdk/core/Azure.Core/src/Shared/RequestContentHelper.cs

Co-authored-by: Christopher Scott <chriscott@hotmail.com>

---------

Co-authored-by: Christopher Scott <chriscott@hotmail.com>

yifanz7 committed:
[ElasticSan] API version 2023-03-01: support snapshots and CMK (#39215)

* ElasticSan snapshots and CMK

* Update changelog to remove empty sections

* Update sourceId format

azure-sdk committed:
Update Generator Version 3.0.0-beta.20231019.2 (#39393)

azure-sdk committed:
Increment package version after release of Azure.Messaging.EventGrid (#39373)

azure-sdk committed:
Increment package version after release of Azure.ResourceManager.NetApp (#39381)

christothes committed:
Prep Azure.Identity.Broker for release (#39375)

jsquire committed:
[CODEOWNERS] Fix Schema Registry (#39396)

The focus of these changes is to fix the Schema Registry service entry to use the special `<NotInRepo>` token rather than a path, allowing our new automation to correctly parse it.

azure-sdk committed:
Increment package version after release of Azure.Identity.Broker (#39395)

abhishesingh-msft committed:
Add Setter to CallInvite to allow pstn to pstn tranfer call (#39397)

azure-sdk commented:
**API change check**

APIView has identified API level changes in this PR and created following API reviews.

[Azure.ResourceManager.Communication](https://apiview.dev/Assemblies/Review/9d869982248d4787aa9ae4dbda26a4b4)

benbp committed:
Bump min tls version to 1.2 for digital twins live test storage account (#39402)

amis92 committed:
fix(monitor): remove newing-up unused object in UseAzureMonitor (#39356)

fixes #38787

benbp committed:
Bump min tls version to 1.2 for digital twins live test event hub (#39405)

* Bump min tls version to 1.2 for digital twins live test storage account

* Bump min tls version to 1.2 for digital twins live test event hub

azure-sdk committed:
Fix update test resources tagging when tags are empty. Bump max limit (#39409)

Co-authored-by: Ben Broderick Phillips <bebroder@microsoft.com>

t-abebchuk committed:
2023 08 15 (#39145)

* sdk generation

* adding tests

* mitigate breaking change

* export API

* all tests pass

* small fix

* csproj version

* changelog

* small fix

* fix release date

* remove unreleased

* updating assets.json to fit the new recordings

* Wei comment fixes

* fixes

* renaming

* recording tag

---------

Co-authored-by: Wei Hu <huwe@microsoft.com>

azure-sdk committed:
Update AutoRest C# version to 3.0.0-beta.20231019.3 (#39416)

gfeitosa-msft committed:
Gfeitosa/increase coverage include int (#39392)

* Remove common package from coverage report

* Added SMS speciffic phone number to test pipeline

* Fix typo in yml file

* Extra tests implemented

* Add a few extra tests for coverage

* Add a few extra tests for coverage

* Remove unecessary usings

* Fix assert exception

* Remove shared core ConnectionString from tests

* Make internal classes visible to test package

* Implement SmsRestClient tests for coverage

* Remove duplicate snippets

* Update code owners for SMS sdk to include only SMS team

* Remove missing user from codeowners file

m-redding committed:
Update CHANGELOG.md (#39406)

jaschrep-msft committed:
Datamovement shares oauth (#39164)

* datamovemnet access to header and audience

* include audience in HttpAuthorization

* tests

* small changes

* rebased and adopted a rename from main

* fix

* exportapi

jaschrep-msft committed:
DataMovement unify TransferAndVerify Part 2 (#39364)

* directory download tests

* unused imprt

JoshLove-msft committed:
Fix deadlettering and exception propagation in Grpc service (#39412)

* Fix deadlettering in Grpc service

* Propagate exception details

* remove project ref

azure-sdk committed:
Increment package version after release of Microsoft.Azure.WebJobs.Extensions.ServiceBus (#39419)

gaagarwalMS committed:
Add Microsoft.NetworkAnalytics RP SDK package (#39240)

jalauzon-msft committed:
[Storage][DataMovement] Add source/destination checkpoint data to Job Plan file (#39411)

amnguye committed:
[Storage] Added Fix for Creating Share File before service copy; Added Data Movement E2E service copy tests (#39440)

* WIP

* WIP - tests written but CopyAuthorization needs to be done first

* Added tests for single sync copy file share; Added minor fixes for creation of file beforehand

* Fix recording

* Fix to share file resource tests

devinlstoen committed:
Prep Azure.Microsoft.Support for release (#39403)

scottaddie committed:
Add EnvironmentCredential to token cache table (#39476)

scottaddie committed:
Replace Azure AD references with Entra (#39477)

azure-sdk committed:
Check az login for target subscription in stress deploy login (#39479)

Co-authored-by: Ben Broderick Phillips <bebroder@microsoft.com>

benbp committed:
Set min tls version for search+storage live tests (#39471)

trrwilson committed:
Update StreamingChatCompletions.cs (#39480)

jalauzon-msft committed:
[Storage][DataMovement] Cleanup after recent changes to checkpointer (#39456)

azure-sdk committed:
Failed Update AutoRest C# version to 3.0.0-beta.20231024.1 (#39463)

azure-sdk committed:
Increment version for kusto releases (#39485)

tg-msft committed:
Adding basic [CallerShouldAudit] support and initial support for Storage and Tables (#39345)

Adding basic [CallerShouldAudit] support and initial annotations of Storage and Tables.

frantran committed:
Update ContainerServiceFleet to 10-15 (#39487)

yifanz7 committed:
[StorageMover] Upgrade to API version 2023-10-01 (#39495)

* StorageMover 2023-10-01

* update version

azure-sdk committed:
Increment package version after release of Azure.ResourceManager.Support (#39488)

azure-sdk committed:
Increment package version after release of Azure.ResourceManager.Peering (#39497)

ArthurMa1978 committed:
Fix tests for Fleet (#39496)

jaschrep-msft committed:
DataMovement unify TransferAndVerify Part 3 (#39401)

* block

* append

* net462 fix

* default timeouts

* test proxy

jaschrep-msft committed:
DataMovement unify TransferAndVerify Part 4 (#39452)

azure-sdk committed:
Update Generator Version 3.0.0-beta.20231025.1 (#39504)

drielenr committed:
Devcenter dataplane TypeSpec preview SDK v2023-04-01  (#39358)

* add tsp location and remove autorest

* SDK generation from TypeSpec

* Update tests and samples reflected by client changes

* Generate snippets

* Generate API updates

* Add breaking changes to CHANGELOG

* Update parameter to delayUntil

chiangvincent committed:
[Event Hubs] Handle cases where desired worker count exceeds int.MaxInt (#39468)

* handle long desired worker count

* adding additional unit test

* use checked approach

* remove unncessary cast

* add unit test

* update comment

* add unit tests

* handle long desired worker count

* adding additional unit test

* use checked approach

* remove unncessary cast

* add unit test

* update comment

* fix merge conflict

* fix concurrency in unit tests

---------

Co-authored-by: Vincent Chiang <vchiang@microsoft.com>

azure-sdk committed:
Update AutoRest C# version to 3.0.0-beta.20231025.3 (#39521)

chiangvincent committed:
[Service Bus] Handle cases where desired worker count exceeds int.MaxInt (#39474)

* handle long desired worker count

* adding additional unit test

* handle overflow

* Revert "adding additional unit test"

This reverts commit a0869c47bb4898acfc7cab3e78b52d1e4bc72dbc.

* Revert "handle overflow"

This reverts commit dba066189f252936a2d21e0d7406cab95e089358.

* Revert "Revert "handle overflow""

This reverts commit bd6bd1d28fc30fb55fe2972cee60aedfcc008821.

* Revert "handle long desired worker count"

This reverts commit e5a8756fbf8fff87409cea415f7c413dd8083bf0.

* add checked block

---------

Co-authored-by: Vincent Chiang <vchiang@microsoft.com>

azure-sdk committed:
Update AutoRest C# version to 3.0.0-beta.20231026.1 (#39526)

* Update Generator Version 3.0.0-beta.20231026.1

* Update SDK codes me_pu_4

* roll back workaround due to `logicalPath` not supported in lro

---------

Co-authored-by: Mingzhe Huang (from Dev Box) <mingzhehuang@microsoft.com>

azure-sdk committed:
Increment package version after release of Azure.ResourceManager.ContainerServiceFleet (#39524)

JoshLove-msft committed:
Sb cross receiver samples (#39514)

* Add samples demonstrating cross receiver settlement

* Fix

* Fix

* fix line break

* Fix sln

* regenerate

* Apply suggestions from code review

Co-authored-by: Jesse Squire <jesse.squire@gmail.com>

---------

Co-authored-by: Jesse Squire <jesse.squire@gmail.com>

JoshLove-msft committed:
Add samples link in README (#39529)

jalauzon-msft committed:
[Storage][DataMovement] Fixes to pause/resume around enumeration (#39486)

m-redding committed:
attributes (#39428)

archerzz committed:
Upgrade `Azure.ClientSdk.Analyzers` and disable enhanced analysis for some SDKs (#39518)

- update the client sdk analyzer to dev build
- add globla config to suppress enhanced analysis rules
- modify ci steps to support suppression by a property

* disable enhanced analysis for backward compatibility

* update `Azure.ClientSdk.Analyzers` to `0.1.1-dev.20231025.2`

---------

Co-authored-by: Mingzhe Huang (from Dev Box) <mingzhehuang@microsoft.com>

mattgotteiner committed:
Update Microsoft.Search to include 2023-11-01 API version (#39326)

* initial regeneration

* update tests

* update generated code

* generated code feedback

* updating recordings

* fix tests

* recording tests

* recording tests

* regenerate code

* updating generated code

* update

* updating tests

* update changelog

* fixing changelog

---------

Co-authored-by: Matt Gotteiner <magottei@microsoft.com>

alec-baird committed:
Update Azure.ResourceManager.DesktopVirtualization to new api version 2023-09-05 (#39025)

azure-sdk committed:
Increment version for elasticsan releases (#39547)

azure-sdk committed:
Increment version for desktopvirtualization releases (#39548)

azure-sdk committed:
Update Generator Version 3.0.0-beta.20231027.1 (#39550)

matthohn-msft committed:
Test updates for 2023-06-01-preview SDK

kagbakpem commented on sdk/communication/Azure.ResourceManager.Communication/tests/ScenarioTests/SuppressionListTests.cs:
```diff
@@ -0,0 +1,211 @@
+            var suppressionLists = await _domainResource.GetSuppressionListResources().GetAllAsync().ToEnumerableAsync();
+
+            Assert.IsNotNull(suppressionLists);
+            Assert.IsTrue(suppressionLists.Count() >= listNames.Length);
```
Why not check for exact count?

matthohn-msft replied:
Updated this scenario to use exact count.

kagbakpem commented on sdk/communication/Azure.ResourceManager.Communication/tests/ScenarioTests/SuppressionListTests.cs:
```diff
@@ -0,0 +1,211 @@
+            var suppressionListAddresses = await suppressionList.GetSuppressionListAddressResources().GetAllAsync().ToEnumerableAsync();
+
+            Assert.IsNotNull(suppressionListAddresses);
+            Assert.IsTrue(suppressionListAddresses.Count() >= addresses.Length);
```
Why not check for exact count?

kagbakpem commented on sdk/communication/Azure.ResourceManager.Communication/tests/ScenarioTests/SuppressionListTests.cs:
```diff
@@ -0,0 +1,211 @@
+            Assert.AreNotEqual(created.Data.FirstName, updated.Value.Data.FirstName);
+            Assert.AreNotEqual(created.Data.LastName, updated.Value.Data.LastName);
+            Assert.AreEqual(created.Data.Notes, updated.Value.Data.Notes);
+        }
```
Can you add one check to check for the total count of the list item after remains the same as before?

kagbakpem commented on sdk/communication/Azure.ResourceManager.Communication/tests/ScenarioTests/SuppressionListTests.cs:
```diff
@@ -0,0 +1,211 @@
+            Assert.AreEqual(created.Data.Notes, updated.Value.Data.Notes);
+        }
+    }
+}
```
Can you also add one test to make sure duplicates are handled properly? List of [a123@abc.com, a234@abc.com, a123@abc.com] results in only adding [a123@abc.com, a234@abc.com]?

matthohn-msft replied:
Will test this with e2e tests instead of the sdk for now.

matthohn-msft committed:
Merge branch 'matthohn-msft-communication-sdk-2023-06-01-preview' of https://github.com/matthohn-msft/azure-sdk-for-net into matthohn-msft-communication-sdk-2023-06-01-preview

ArthurMa1978 commented:
@matthohn-msft please re-record all tests by following https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/resourcemanager/Azure.ResourceManager/docs/TestGuide.md

matthohn-msft committed:
Updating tests and addressing PR comments.

matthohn-msft committed:
Comment out RecordedTestMode.Record

archerzz commented on sdk/communication/Azure.ResourceManager.Communication/src/Generated/SuppressionListResourceData.cs:
```diff
@@ -0,0 +1,50 @@
+        /// <summary> The the name of the suppression list. This value must match one of the valid sender usernames of the sending domain. </summary>
+        public string ListName { get; set; }
+        /// <summary> The date the resource was last updated. </summary>
+        public string LastUpdatedTimeStamp { get; }
```
Could this be a type of `DateTimeOffset`? See an example [here](https://github.com/Azure/azure-sdk-for-net/blob/e4fce1607d65d709b19c81cc78969cd08b804244/sdk/hdinsight/Azure.ResourceManager.HDInsight/src/autorest.md?plain=1#L146).

archerzz replied:
same as `CreatedTimeStamp` below

matthohn-msft replied:
Please take a look at the changes.  Autorest did not detect these changes

  SuppressionListProperties.createdTimeStamp: -|date-time
  SuppressionListProperties.lastUpdatedTimeStamp: -|date-time

archerzz replied:
> Please take a look at the changes. Autorest did not detect these changes
> 
> SuppressionListProperties.createdTimeStamp: -|date-time SuppressionListProperties.lastUpdatedTimeStamp: -|date-time

@matthohn-msft It should be:

```
SuppressionListResource.properties.createdTimeStamp: -|date-time
SuppressionListResource.properties.lastUpdatedTimeStamp: -|date-time
```

I've commented in the changes in `autorest.md`.

matthohn-msft committed:
Merge branch 'Azure:main' into matthohn-msft-communication-sdk-2023-06-01-preview

matthohn-msft committed:
update autorest

archerzz commented on sdk/communication/Azure.ResourceManager.Communication/src/autorest.md:
```diff
@@ -74,6 +74,8 @@ rename-mapping:
   DnsRecord: VerificationDnsRecord
   DomainsProvisioningState: DomainProvisioningState
   ProvisioningState: CommunicationServiceProvisioningState
+  SuppressionListProperties.createdTimeStamp: -|date-time
```
```suggestion
  SuppressionListResource.properties.createdTimeStamp: -|date-time
```

Same as `SuppressionListProperties.lastUpdatedTimeStamp: -|date-time`

archerzz reviewed (commented):
Some codes are not up to date. Need to rebuild the codes.

matthohn-msft committed:
use datetimeoffset

archerzz reviewed (approved):
@matthohn-msft  The changes look fine. The remaining issues are:
- Update the codes and API signature. See the error logs for details.
- Fix the test case issues.

matthohn-msft committed:
Updating tests and assets

matthohn-msft committed:
Merge branch 'main' into matthohn-msft-communication-sdk-2023-06-01-preview

archerzz reviewed (commented):
@matthohn-msft You need to regen the codes, since there are some changes in the generator used in `main` branch.

matthohn-msft committed:
rebuilding package to pick up changes from main

matthohn-msft committed:
Disable record mode

matthohn-msft commented:
> @matthohn-msft You need to regen the codes, since there are some changes in the generator used in `main` branch.

@archerzz code has been regenerated.

matthohn-msft committed:
Update version for acs resources

archerzz committed:
update api signature

archerzz reviewed (commented):
@matthohn-msft 
If you are going to release a new version, don't forget to update `CHANGELOG.md`. See [here](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/78/Mgmt-.NET-SDK-release-process?anchor=signing-and-publishing-nuget-packges) for details.

matthohn-msft commented on sdk/communication/Azure.ResourceManager.Communication/tests/ScenarioTests/SuppressionListTests.cs:
```diff
@@ -0,0 +1,286 @@
+
+                // // todo: investigate why pageSize is not working.
+                // Assert.AreEqual(pageSize, page.Values.Count);
+
```
Determined the issue, it is in the service side. I will prepare fix @kagbakpem 
When a response has no more pages, I should be returning a null nextLink value. Because the nextLink is never null, the sdk framework will never find an end to a collection when calling GetAllAsync().

matthohn-msft committed:
Updating changelog.md

matthohn-msft commented:
/azp run

matthohn-msft commented:
> @matthohn-msft 
> If you are going to release a new version, don't forget to update `CHANGELOG.md`. See [here](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/78/Mgmt-.NET-SDK-release-process?anchor=signing-and-publishing-nuget-packges) for details.

Updates the changeling, please take a look
